docker-compose.dist.yml

```diff
@@ -7,50 +7,31 @@ services:
         container_name: koillection
         build:
             dockerfile: Dockerfile.dev
-        restart: always
         ports:
             - 81:80
         environment:
-            - APP_ENV=prod
-            - APP_DEBUG=0
-            - DB_DRIVER=pdo_pgsql
+            - APP_ENV=dev
+            - APP_DEBUG=1
+            - DB_DRIVER=pdo_mysql
             - DB_NAME=koillection
-            - DB_HOST=postgresql
-            - DB_PORT=5432
-            - DB_USER=postgres
+            - DB_HOST=mysql
+            - DB_PORT=3306
+            - DB_USER=root
             - DB_PASSWORD=password
             - DB_VERSION=15
             - PHP_TZ=Europe/Paris
-            - HTTPS_ENABLED=1
+            - HTTPS_ENABLED=0
         depends_on:
             - mysql
-            - postgresql
         volumes:
             - "./docker/volumes/koillection/public/uploads:/var/www/koillection/public/uploads"
             - ./:/var/www/koillection
 
-    postgresql:
-        container_name: postgresql
-        image: postgres:15
-        environment:
-            - POSTGRES_DB=koillection
-            - POSTGRES_USER=postgres
-            - POSTGRES_PASSWORD=password
-            - POSTGRES_HOST_AUTH_METHOD=trust
-        ports:
-            - 5432:5432
-        volumes:
-            - "./docker/volumes/postgresql:/var/lib/postgresql/data"
-
     mysql:
-        container_name: mysql
         image: mysql:latest
-        command: --default-authentication-plugin=mysql_native_password
+        container_name: mysql
         environment:
-            - MYSQL_ROOT_PASSWORD=password
             - MYSQL_DATABASE=koillection
-
-        ports:
-            - 3306:3306
+            - MYSQL_ROOT_PASSWORD=password
```

Comment:
The order doesn't matter here ;)

Reply:
Aligned to docker-compose.yml for production and dev in upstream. Will be anyways when fork sync happens.